build_deps/build_pip_pkg.sh

```diff
@@ -78,7 +78,7 @@ function main() {
   BUILD_CMD="setup.py bdist_wheel --platlib-patch"
   if is_macos; then
     if [[ x"$(arch)" == x"arm64" ]]; then
-      BUILD_CMD="${BUILD_CMD} --plat-name macosx_11_0_arm64"
+      BUILD_CMD="${BUILD_CMD} --plat-name macosx_12_0_arm64"
```

Comment:
Are we not going to distribute macosx_11 packages anymore?

I see that `tensorflow-macosx` wheels are still on `macosx_11_0`:
https://pypi.org/project/tensorflow-macos/2.6.0/#files

Reply:
This is build from source issue, so only local wheel files. We seem to have only a minimal macOS version support, but if I understand the platform correctly, we may need to change the `build_pip_pkg.sh` file to instead check for either x86 or M1 and OS version.

Reply:
I don't think it is only local as we are using the same in Action:

https://github.com/tensorflow/addons/blob/master/.github/workflows/make_wheel_macOS_arm64.sh#L27

https://github.com/tensorflow/addons/blob/41eaa27d49025c02bfe9520d5e63e1f01a782ddf/BUILD#L3